### ISSUE TITLE
GodMode: suggestion - SmartGPT functionality

### ISSUE THREAD

**Issue by jacob1983** [smol-ai/menubar#132]
Suggestion: Implementing a feature that utilizes [SmartGPT](https://www.youtube.com/watch?v=wVzuvf9D9BU) utilizing various models.

Expected Behavior: All outputs from the models should be directed to GPT-4, which will then assess and determine the best response, as depicted in the image.
![be436c_e58ba51be01c4699991e2ce4eeba1d03~mv2](https://github.com/smol-ai/menubar/assets/2161493/97e26b1a-c6be-4570-a558-83c656af21b6)

**Prompts Frameworks:** (from video)

_Answer: Let's work this out in a step by step way to be sure we have the right answer

You are a researcher tasked with investigating the X response options provided. List the flaws and faulty logic of each answer option. Let's work this out in a step by step way to be sure we have all the errors:

You are a resolver tasked with 1) finding which of the X answer options the researcher thought was best 2) improving that answer, and 3) Printing the improved answer in full. Let's work this out in a step by step way to be sure we have the right answer:_

**Comment by swyxio** [comment]
yes, agree, we have internally talked about something like this before.

**Comment by PhillipRt** [comment]
Chatall kind of solved that. They judt buld a big prompt and send it to chatgpt or Claude. Results seem pretty convincing, also some prompt engineering might be required
